napari/_qt/qt_viewer.py

```diff
@@ -278,6 +280,15 @@ def dragEnterEvent(self, event):
         else:
             event.ignore()
 
+    def _open_files(self):
```

Comment:
can you move `_open_files` and `_add_files` back up to where they were before, it will make the diff smaller and easier to understand the changes